tests/fixtures/class2tree_classification_dup_call.yml

```diff
@@ -1,36 +1,36 @@
 http_interactions:
 - request:
     method: get
-    uri: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=taxonomy&term=Mus%2Bmusculus&api_key=<<entrez_api_token>>
```

Comment:
looks like you aren't using an entrez API key? Do you have one?  If you don't, just don't commit changes to these fixture files since I don't think they should be changing anyway

Reply:
No, I am not using the entrez API key. Just like last time, I had to remove them so that the test can be passed. So you mean, I just need to change it on my end and not submit the changes to GitHub? If so, I've reset them. Hope that's fine.

Reply:
Yes, just don't commit changes to any fixtures